beam/core/src/main/java/cz/o2/proxima/beam/core/io/StreamElementCoder.java

```diff
@@ -101,6 +106,7 @@ public StreamElement decode(InputStream inStream) throws IOException {
                 () -> new IOException(String.format("Unable to find entity [%s].", entityName)));
 
     final String uuid = input.readUTF();
+    final long sequentialId = uuid.length() == 0 ? input.readLong() : -1L;
```

Comment:
@je-ik This breaks us internally, we've were using empty `uuid` inside beam pipelines

Reply:
Uh, you should not ... can we create `StreamElementWithoutUUIDCoder` for that?

Reply:
We can probably work around that in user code, I'll keep you updated